crates/rpc/rpc-api/src/eth.rs

```diff
@@ -18,6 +18,15 @@ pub trait EthApi {
     #[method(name = "protocolVersion")]
     async fn protocol_version(&self) -> RpcResult<U64>;
 
+    /// Executes complex RPC calls to Ethereum nodes
+    #[method(name = "ethMulticallV1")]
```

Comment:
jsonrpc method names are case sensitive and eth prefix is already specified on the trait 
```suggestion
    #[method(name = "multicallV1")]
```